src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,58 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import { useEffect, useState } from 'react';
+import { useMyContext } from '../../context/myContext';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+export const TodoFilter: React.FC = () => {
+  const [qwerty, setQwerty] = useState<string>('');
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+  const { searchTodo, handleCompletedTodos } = useMyContext();
+
+  useEffect(() => {
+    searchTodo(qwerty);
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [qwerty]);
```

Comment:
you have a place where you change your query, it's Input's `onChange` 
put `searchTodo` there and there will be no need for a `useEffect`